code/modules/surgery/organs/organ_external.dm

```diff
@@ -138,6 +138,29 @@
 	replaced(H)
 	return 1
 
+/obj/item/organ/external/attack(mob/living/carbon/C, mob/user) // Limb re-attachment
+	if(ishuman(C)) // Is it human?
+		var/mob/living/carbon/human/H = C
+		if(H.species && H.species.species_traits && EASYLIMBATTACHMENT in H.species.species_traits) // Can it attach limbs like this?
+			if((H.species.reagent_tag == PROCESS_SYN && status == ORGAN_ROBOT) || (H.species.reagent_tag = PROCESS_ORG && status != ORGAN_ROBOT)) // Robotic with Robotic, organic with organic.
```

Comment:
`reagent_tag = PROCESS_ORG` :thinking: 
(should be ==)

Reply:
I also don't think that checking their reagent processing is the best way of handling these checks, since this has absolutely nothing to do with reagent metabolization.

Reply:
This is the only way I can think of that flags "Organic" or "Synthetic" in a general sense.

I'll fix up what you commented on today or tomorrow, tiger.

Reply:
why not something like `if((is_species(A, "Machine") && status == ORGAN_ROBOT) || (!is_species(A, "Machine") && status != ORGAN_ROBOT)`

Reply:
Honestly I'm not sure if that's less snowflakey or more snowflakey. I'll leave that up to the maintainers I guess?

Reply:
It's better like this.